### DISCUSSION TITLE
Deploy to azure and create endpoint for the prediction pipeline only

### DISCUSSION THREAD

**Discussion by GeorgeDrakoulas** [kedro-org/kedro#3479] [Idea]
Dear all,

I wish you a Happy New Year!

I have created the framework of a regression model, using Kedro.
The framework consists of 3 pipelines.
1. Data_processing
2. Data_science
3. Prediction

In pipeline 1, I receive and pre-process the data.
In pipeline 2, the machine learning model is trained and tested.
In pipeline 3, parts from the pipeline 1 and 2 are utilized so as to make the prediction using the ML model for a new set that is pre-processed accordingly.

Now, I need to deploy to azure ML only the pipeline 3: prediction phase, and create the endpoint for that.
Is tha possible using Kedro? and if yes, how?

Thank you very much in advance.

Regards,
George

**Comment by astrojuanlu** [comment]
Hi @GeorgeDrakoulas, happy new year to you too!

Please have a look at https://github.com/getindata/kedro-azureml/

Also, these Discussions are not very well monitored, we might disable them in the near future. To get quicker support, please use our Slack workspace: http://slack.kedro.org/
